### PR TITLE
[improvement]replace margin with padding to improve touch experience …

### PR DESCRIPTION
**Pull request by inooNgt**
replace margin with padding to improve touch experience in radio label

### Before submitting a pull request, please make sure the following is done:

1. Fork [the repository](https://github.com/youzan/vant) and create your branch from `dev`.
2. Follow the [contributing guide](https://github.com/youzan/vant/blob/dev/.github/CONTRIBUTING.md) to setup your develop environment.
2. If you've added code that should be tested, add tests.
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).

#### Title Format

[bugfix / new feature / improvement] ComponentName：content

Example：
* [bugfix] Button: color prop not work
* [new feature] Button: add color prop
* [improvement] Button: optimize performance

### CODE REVIEW

**Comment by chenjiahan** [comment]
请问改动的原因是？

**Comment by inooNgt** [comment]
> 请问改动的原因是？

用margin时，点到margin的区域不会触发选择事件，用padding可以触发，体验更好。

File: src/radio/index.less [hunk 1]
```diff
@@ -54,7 +54,7 @@
   }
 
   &__label {
-    margin-left: @radio-label-margin;
+    padding-left: @radio-label-padding;
```
Comment (chenjiahan) [inline src/radio/index.less]: `label--left` 的情况也需要改成 padding

**Comment by chenjiahan** [comment]
明白 👌

**Pull request closed without merging**